### PR TITLE
Set Docker's default log level to warning to reduce log spam.

### PR DESCRIPTION
**Pull request by a-robinson**
This may be controversial, but continuing discussion from #15590 here.

@kubernetes/goog-node @yujuhong @vishh @dchen1107 @jimmidyson

### CODE REVIEW

File: cluster/saltbase/salt/docker/docker-defaults [hunk 1]
```diff
@@ -1,3 +1,3 @@
 {% set grains_opts = grains.get('docker_opts', '') -%}
-DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false"
+DOCKER_OPTS="{{grains_opts}} --bridge=cbr0 --iptables=false --ip-masq=false --log-level=warning"
```
Comment (jimmidyson) [inline cluster/saltbase/salt/docker/docker-defaults]: Shouldn't that be `warn`, not `warning`?

Comment (a-robinson) [inline cluster/saltbase/salt/docker/docker-defaults]: Both work, judging by the code:
https://github.com/docker/docker/blob/729c9a97822ebee2c978a322d37060454af6bc66/vendor/src/github.com/Sirupsen/logrus/logrus.go#L43

Comment (dchen1107) [inline cluster/saltbase/salt/docker/docker-defaults]: yes, it should be warn, not warning.

Comment (dchen1107) [inline cluster/saltbase/salt/docker/docker-defaults]: Here is the man page: 

https://github.com/docker/docker/blob/c45ad0b02d1cc1f88aaabab18aed27aeb20b9e7a/man/docker.1.md#L46

Comment (a-robinson) [inline cluster/saltbase/salt/docker/docker-defaults]: Ack, updated.

**Comment by jimmidyson** [comment]
:+1:

**Comment by yujuhong** [comment]
lgtm

**Comment by a-robinson** [comment]
Also just an FYI of a relevant message I saw in the logs when testing this out:
`Warning: '-d' is deprecated, it will be removed soon. See usage.`

**Comment by dchen1107** [comment]
The only concern I had is that I observed that before some useful logging related to docker exec is info level, not warn level. But I guess this is the tradeoff we have to make. 

LGTM

**Comment by a-robinson** [comment]
@k8s-bot please test this

**Comment by wojtek-t** [comment]
@k8s-bot test this please

**Pull request merged**